src/integrations.md

```diff
@@ -5,3 +5,4 @@
 - [Gitea Actions runner](https://gitea.com/gitea/act_runner)
 - [github-act-runner](https://github.com/ChristopherHX/github-act-runner)
 - [Gradle Act plugin](https://github.com/pshevche/gradle-act-plugin)
+- [GitHub Local Actions](https://marketplace.visualstudio.com/items?itemName=SanjulaGanepola.github-local-actions)
```

Comment:
```suggestion
- [GitHub Local Actions for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=SanjulaGanepola.github-local-actions)
```

Reply:
@cplee I see what you mean now. Can I go ahead with this change but keep it at the previous link I had?

Reply:
Whatever link you want works for me - so long as the text of the link is clear what it is.

Reply:
Great, I updated to the previous link